scripts/aws/cfn/microservice.yml

```diff
@@ -91,85 +91,387 @@ Parameters:
+    Description: 'Threshold for autoscaling'
+    Default: 1000
+
+  KinesisStreamArn:
+    Type: String
+    Description: 'Arn of the kinesis stream where paper event enrichments are stored'
+
 Resources:
 
-#  PaperEventEnricherMicroservice:
-#    Type: AWS::CloudFormation::Stack
-#    Properties:
-#      TemplateURL: !Sub "${TemplateBucketBaseUrl}/fragments/ecs-service.yaml"
-#      Parameters:
-#        MicroServiceUniqueName: !Sub '${ProjectName}-template-ms-be'
-#        CpuValue: 512
-#        MemoryAmount: 2GB
-#        AlarmSNSTopicArn: !Ref AlarmSNSTopicArn
-#        ContainerImageURI: !Sub '${ContainerImageUri}'
-#        ContainerEnvEntry1: !Sub 'AWS_REGIONCODE=${AWS::Region}'
-#        ContainerEnvEntry2: !Sub 'PN_PAPER_EVENT_ENRICHER_DAO_PAPEREVENTENRICHMENTTABLE=${PaperEventEnrichmentTableName}'
-#        ContainerEnvEntry3: !Sub 'PN_PAPER_EVENT_ENRICHER_TOPICS_GATEWAYEVENTS=${PaperArchivesQueueName},${PaperEventEnrichmentInputQueueName}'
-#        ContainerEnvEntry4: !Sub 'PN_PAPER_EVENT_ENRICHER_SQS_PAPERARCHIVESQUEUENAME=${PaperArchivesQueueName}'
-#        ContainerEnvEntry5: !Sub 'PN_PAPER_EVENT_ENRICHER_SQS_PAPEREVENTENRICHMENTINPUTQUEUENAME=${PaperEventEnrichmentInputQueueName}'
-#        ContainerEnvEntry6: !Sub 'PN_PAPER_EVENT_ENRICHER_SAFESTORAGE_BASEURL=http://${ApplicationLoadBalancerDomain}:8080'
-#        ContainerEnvEntry7: !Sub 'PN_PAPER_EVENT_ENRICHER_CXID=${PaperEventEnrichmentCxId}'
-#        MappedPaths: '/*'
-#        ECSClusterName: !Ref ECSClusterName
-#        Subnets: !Ref SubnetsIds
-#        VpcId: !Ref VpcId
-#        EcsDefaultSecurityGroup: !Ref EcsDefaultSecurityGroup
-#        LoadBalancerListenerArn: !Ref ApplicationLoadBalancerListenerArn
-#        LoadbalancerRulePriority: !Ref MicroserviceNumber
-#        CloudwatchLogKinesisStreamArn: !Ref LogsKinesisSourceStreamArn
-#        TaskRoleManagedPolicyArn: !Ref PaperEventEnricherMicroserviceTaskManagedPolicy
-#        EcsLogGroup: !Ref EcsLogGroup
-#        LogAlarmStrategyV1: !Ref LogAlarmStrategy
-
-#  PaperEventEnricherMicroserviceTaskManagedPolicy:
-#    Type: AWS::IAM::ManagedPolicy
-#    Properties:
-#      PolicyDocument:
-#        Version: 2012-10-17
-#        Statement:
-#          - Effect: Allow
-#            Action:
-#              - dynamodb:GetItem
-#              - dynamodb:PutItem
-#              - dynamodb:DeleteItem
-#              - dynamodb:Query
-#              - dynamodb:UpdateItem
-#            Resource:
-#              - !Sub "${PaperEventEnrichmentTableArn}"
-#              - !Sub "${PaperEventEnrichmentTableArn}/*"
-#          - Effect: Allow
-#            Action:
-#              - sqs:SendMessage
-#              - sqs:GetQueueUrl
-#              - sqs:ChangeMessageVisibility
-#              - sqs:DeleteMessage
-#              - sqs:GetQueueAttributes
-#              - sqs:ReceiveMessage
-#            Resource:
-#              - !Ref PaperArchivesQueueARN
-
-#  # CloudWatch dashboard
+  PaperEventEnricherMicroservice:
```

Comment:
@flaminiaScarciofolo rispetto a https://github.com/pagopa/pn-kafka-bridge/blob/develop/scripts/aws/cfn/microservice.yml , ad esempio, mancano alcuni parametri come min Max tasks number, i dati sul mount efs.

Reply:
Il mountEfs è necessario? ho aggiunto gli altri parametri.

Reply:
Si, i due parametri per il mount efs servono perché consentono di attivare il memory dump e salvare l'output in maniera persistente in caso di problemi.